src/pages/register/index.tsx

```diff
@@ -1,18 +1,73 @@
+import { useState } from 'react'
+
+import { useRouter } from 'next/router'
+
 import { Title } from 'styles/pages'
 
 import { Container } from 'components/Container'
 import { Field } from 'components/Field'
 import { Button } from 'components/Button'
 
+import { isValidEmail, isValidPassword } from '../../utils/authValidations'
```

Comment:
If the utils directory is at the root of the src directory, prefer calling it directly `utils/...` instead of `../../utils/...`